src/frontend/apps/impress/src/features/docs/doc-editor/components/DocEditor.tsx

```diff
@@ -58,10 +58,10 @@ export const DocEditor = ({ doc }: DocEditorProps) => {
       )}
       <Box
         $background={colorsTokens()['primary-bg']}
-        $height="100%"
         $direction="row"
+        $width="100%"
```

Comment:
Not sure if it is because of that, it goes out of the screen:
![image](https://github.com/user-attachments/assets/f7b529bd-0a5f-42b3-b608-6e76fc44a547)